### PR TITLE
Added predicate to find all highlights, such that it only shows for current user.

### PR DESCRIPTION
**Pull request by Dendrobyte**
`where` has been included in the search query

🙃

### CODE REVIEW

**Comment by ghost** [comment]
`forbiddenHighlightsChannels` should probably be added to config.json.example. Also is there a reason as to why in the initial PR there's storageData.db and storageDatas.db? Can one of these be removed?

File: eventActions/highlightActions.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 const Highlights = require('../databaseFiles/highlightsTable.js');
 const Discord = require('discord.js');
+const config = require('../config.json');
 
 class highlightActions {
 
 	// Method to call to check a message for a highlighted message
 	static async checkForHighlight(client, message){
 		if(message.content.charAt(0) === '!') return; // Ensure commands aren't caught
```
Comment (Tekrus) [inline eventActions/highlightActions.js]: This should use the prefix from the config file

**Review by Tekrus — CHANGES_REQUESTED** [review]

File: eventActions/highlightActions.js (hunk 1 above)
Comment (ghost) [reply eventActions/highlightActions.js]: I'm confused on what this means 😕 like making sure it uses the prefix value? Isn't it default used for any commands?

**Comment by ghost** [comment]
Adjustment to my previous change request: now I see that storageData.db is being deleted so 👍 but another DB is also there, and databaseFiles contains a 3rd DB file, can we chat in #r&d about how we wanna structure this? Might be helpful.

**Review by Tekrus — APPROVED** [review]
My comments are fixed, requesting a re-review from @cosmiccdev

**Review by Tekrus — CHANGES_REQUESTED** [review]
Actually, I was too quick about that, you forgot to update the example config file with the newly added forbiddenHighlightChannels array

**Comment by ghost** [comment]
Yeah, once example config file is fixed I'll approve as well.

**Comment by Dendrobyte** [comment]
Guess I forgot to save it or something, was sure I had pushed it.

Sorry again about delaying this further.

**Review by ghost — APPROVED** [review]

**Review by Tekrus — APPROVED** [review]

**Pull request merged**